CHANGES.md

```diff
@@ -1,3 +1,18 @@
+## edge-19.4.1
+
+* CLI
+  * Introduced an `upgrade` command! This allows an existing Linkerd Control Plane to be reinstalled or reconfigured
```

Comment:
Is there a pithy summary of why/when you would need this command? Not critical but would be nice.

Reply:
Also "control plane" is better lower-cased

Reply:
I will add: "it is particularly useful for automatically reusing flags set in the previous `install` or `upgrade`"